lib/src/base-styles/_nx-lists.scss

```diff
@@ -128,109 +120,65 @@
  * Action or Clickable Lists
  */
 .nx-list--clickable {
-  .nx-list__title {
-    color: $nx-text-color;
-  }
-
   .nx-list__item {
-    @include semi-bold();
-
-    color: $nx-text-color-light;
-
     &.selected {
-      background-color: $nx-grey-3;
-      color: $nx-text-color;
-      pointer-events: none;
-    }
+      background-color: #d3e6ff;
 
-    &:not(.nx-list__item--empty):not(.nx-disabled):not([disabled]) {
-      &:hover, &:focus {
-        background-color: $nx-color-blue-96;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
-        color: $nx-text-color;
-        cursor: pointer;
-        outline-style: none; // removes :focus outline
-
-        .fa-chevron-right {
-          color: $nx-color-blue-41;
-        }
-      }
-
-      &:active {
-        background-color: $nx-grey-1;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
+      &:hover {
+        background-color: #d3e6ff;
       }
     }
 
-    &.nx-list__item--empty {
-      @include regular();
-
-      border-top: $nx-border; // assume it's always the first item
-      color: $nx-text-color-light;
+    &:hover {
+      background-color: #f1f3fa;
+      cursor: pointer;
+      outline-style: none; // removes :focus outline
     }
 
-    &.nx-error {
-      color: $nx-text-color-error;
+    &:active {
+      background-color: #d3e6ff;
+    }
 
-      &:not(.nx-disabled):not([disabled]) {
-        border: $nx-border-error;
 
-        &:hover, &:focus, &:active {
-          background-color: $nx-fail-color-bg;
-          border: $nx-border-error;
-          color: $nx-text-color-error;
-          cursor: default;
-        }
-      }
-    }
+    &.nx-list__item--empty {
+      @include regular();
 
-    .nx-list__subtext {
-      pointer-events: none; // Fix for IE11 :active state display
+      color: $nx-text-color-light;
     }
 
-    &.nx-list__item--disabled {
+    &.disabled {
       color: $nx-text-color-disabled;
-      cursor: default;
 
       &:hover {
-        border-top: 1px solid transparent;
-        border-bottom: $nx-border-row;
+        background-color: transparent;
+        cursor: default;
+      }
+
+      > .nx-chevron {
+        display: none;
       }
     }
-  }
 
-  .fa-chevron-right {
-    margin-right: 3px;
-    margin-top: 1px;
+    > .nx-chevron {
```

Comment:
I think the font-size of this element needs to be adjusted.  The visible glyph of the chevron is 13px tall in zeplin.  In the implementation however, the `<path>` element that makes up the visible glyph is only 10px tall.

Reply:
> I think the font-size of this element needs to be adjusted. The visible glyph of the chevron is 13px tall in zeplin. In the implementation however, the `<path>` element that makes up the visible glyph is only 10px tall.

Some custom tweaking here: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/b9593bb99ff1f7cc9785692b149281df1f97de79

Reply:
> Some custom tweaking here: b9593bb

Seems too big now.  15.6x10 instead of the expected 13x8.  Also if I'm not mistaken the correct way to set the size of font awesome icons is with `font-size`.

Reply:
> > Some custom tweaking here: [b9593bb](https://github.com/sonatype/sonatype-react-shared-components/commit/b9593bb99ff1f7cc9785692b149281df1f97de79)
> 
> Seems too big now. 15.6x10 instead of the expected 13x8. Also if I'm not mistaken the correct way to set the size of font awesome icons is with `font-size`.

https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/e57ab0da139398c2561b16f90a4e9976f7da0c73